data/campaigns/Eastern_Invasion/scenarios/05_Northern_Outpost.cfg

```diff
@@ -193,6 +193,30 @@
     {PLACE_IMAGE items/bones.png 6 31}
     {PLACE_IMAGE scenery/blood-1.png 7 31}
 
+#define CHANGE_VOLUME VOLUME DELAY
+    [volume]
+        music={VOLUME}
+    [/volume]
+    [delay]
+    time={DELAY}
```

Comment:
Bad indent here, see the [CI logs](https://github.com/wesnoth/wesnoth/actions/runs/8271100101/job/22630101946#step:6:23).